src/Entity/BaseEntityManager.php

```diff
@@ -47,6 +48,11 @@ public function getEntityManager()
     {
         return $this->getObjectManager();
     }
+
+    final protected function getRepository(): EntityRepository
```

Comment:
won't this be a bc break if someone already has this method and extended this class?

Reply:
https://xkcd.com/1172/